mods/content/polaris/datum/locations.dm

```diff
@@ -6,24 +170,347 @@
+	centuries of welcoming alien and positronic migrants and refugees. Many of the systems within the Golden Crescent \
+	seceded during the 23rd century before being absorbed back into SolGov, and they maintain a strong independent streak."
+	capital = "Vir"
+	ruling_body = "Golden Crescent Regional Government"
+	distance = null //not one location
+
+//
+// Sagittarius Heights
+//
+
+//
+// New Ohio
+/decl/background_detail/location/franklin
+	name = "Franklin, New Ohio"
+	uid = "location_franklin"
+	description = "The jewel of New Ohio and the gateway to the Sagittarius Heights, \
+	Franklin is known for three things: tourism, aquaculture, and hosting the Sagittarian branch offices of many a trans-stellar corporation. \
+	Even after the secession of the Five Arrows, the majority of traffic between the Heights and the core systems passes through New Ohio, \
+	and Franklin is a common stop for passengers who want to spend a bit of time on a garden world while they can."
+	ruling_body = "Solar Confederate Government"
+	distance = null //unknown
+	subversive_potential = 0.9
+
+
+/decl/background_detail/location/wright
+	name = "Wright, New Ohio"
+	uid = "location_wright"
+	description = "Formerly the industrial hub of New Ohio and a major exporter to the whole of the Sagittarius Heights, \
+	Wright has fallen on hard times in the post-Incursion world. With no money, far fewer trade partners, \
+	and many of its smaller stations still abandoned or in ruins, Wright is a shell of its former self."
+	ruling_body = "Solar Confederate Government"
+	distance = null //unknown
+	economic_power = 0.7 //super space detroit. maybe it should be 0.8 or 0.75? I am overthinking it.
+
+
+//
+// Five Arrows
+/decl/background_detail/location/sidhe
+	uid = "location_sidhe"
+	name = "Sidhe"
+	description = "Sidhe hosts an eclectic mix of humans, skrell, and positronics, co-existing in uneasy harmony for hundreds of years. \
+	Formerly controlled by the for-profit Daoine Institute, one of the largest technical schools in SolGov, \
+	using a Skrellian academic model of governance, the system has since seen a popular revolution and has seceded from SolGov. \
+	Today, Sidhe is seen as one of the Five Arrows' most militant supporters and as a case study in human-skrellian cultural integration."
+	local_government = "Sidhe Reconstruction Government"
+	ruling_body = "Five Arrows"
+	economic_power = 0.7
+	subversive_potential = 1.1
+	distance = "27.3 ly"
+
+/decl/background_detail/location/mahimahi
+	uid = "location_mahimahi"
+	name = "Mahi-Mahi, Mahi Mahi"  //dash space
+	description = "A planet split into two distinct spheres of power, Mahi-Mahi hosts an ongoing struggle between \
+	the dichotomous interests of its metropolitan centres and their surrounding rural populations."
+	local_government = "Mahi-Mahi Planetary Government"
+	ruling_body = "Five Arrows"
+	distance = "21.6 ly"
+
+
+//New Seoul goes here
```

Comment:
It's a complete stub but this should probably be here as a location even if it's not in as a culture, since "locations" are used for some other random generation and New Seoul is fairly important politically
https://ss13polaris.com/index.php?title=New_Seoul